### PR TITLE
Update mongoproc to 3.1,3.1.1

### PR DESCRIPTION
**Pull request by core-code**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/mongoproc.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'mongoproc' do
-  version '3.1'
-  sha256 'cc4d2dfc94ae8df014ac0e317663e074e5d128c61df851e19da0fec9912d2a0e'
+  version '3.1,3.1.1'
```
Comment (reitermarkus) [inline Casks/mongoproc.rb]: Why not just use `3.1.1`?

Comment (core-code) [reply Casks/mongoproc.rb]: how would you make sure it uses just 3.1 before the last slash?

Comment (reitermarkus) [reply Casks/mongoproc.rb]: `version.major_minor`

Comment (core-code) [reply Casks/mongoproc.rb]: ah sorry forgot about that option ^^

**Pull request merged**